plone/namedfile/browser.py

```diff
@@ -8,6 +9,13 @@
 from zope.publisher.interfaces import IPublishTraverse
 from zope.publisher.interfaces import NotFound
 
+try:
```

Comment:
This violates our coding style guide: https://docs.plone.org/develop/styleguide/python.html#about-imports 

please use instead: 


    try:    
        pkg_resources.get_distribution('plone.app.blob')
        if pkg_resources.get_distribution('plone.dexterity').version >(x, y, z):
            from plone.app.blob.download import handleRequestRange
            from plone.app.blob.iterators import BlobStreamIterator
            HAS_PAB = True
    except pkg_resources.DistributionNotFound:
        HAS_PAB = False